packages/glimmer-runtime/lib/dom.ts

```diff
@@ -82,7 +83,7 @@ class DOMHelper {
   }
 
   insertHTMLBefore(parent: HTMLElement, nextSibling: Node, html: string): Bounds {
-    let prev = nextSibling && nextSibling.previousSibling;
```

Comment:
When inserting into an element that already has children with `nextSibling=null`, the current `lastChild` of the parent should be considered the `prev` element. This fixed issues raise in tests below for `<svg></svg>{{{name}}}` on rerender. The bounds incorrectly included the `<svg>` tag (because `let first = prev ? prev.nextSibling : parent.firstChild;` at the end of this function), and thus on re-render the `<svg>` tag was being destroyed.